types/d3-scale-chromatic/index.d.ts

```diff
@@ -45,173 +48,269 @@ export const schemeSet3: string[];
  */
 export function interpolatePRGn(value: number): string;
+
+export const schemePRGn: string[][];
+
 /**
  * Given a number t in the range [0,1], returns the corresponding color from the “PiYG” diverging color scheme represented as an RGB string.
  *
  * @param value Number in the range [0, 1].
  */
 export function interpolatePiYG(value: number): string;
+
+export const schemePiYG: string[][];
+
 /**
  * Given a number t in the range [0,1], returns the corresponding color from the “PuOr” diverging color scheme represented as an RGB string.
  *
  * @param value Number in the range [0, 1].
  */
 export function interpolatePuOr(value: number): string;
+
+export const schemePuOr: string[][];
+
 /**
  * Given a number t in the range [0,1], returns the corresponding color from the “RdBu” diverging color scheme represented as an RGB string.
  *
  * @param value Number in the range [0, 1].
  */
 export function interpolateRdBu(value: number): string;
+
+export const schemeRdBu: string[][];
+
 /**
  * Given a number t in the range [0,1], returns the corresponding color from the “RdGy” diverging color scheme represented as an RGB string.
  *
  * @param value Number in the range [0, 1].
  */
 export function interpolateRdGy(value: number): string;
+
+export const schemeRdGy: string[][];
+
 /**
  * Given a number t in the range [0,1], returns the corresponding color from the “RdYlBu” diverging color scheme represented as an RGB string.
  *
  * @param value Number in the range [0, 1].
  */
 export function interpolateRdYlBu(value: number): string;
+
+export const schemeRdYlBu: string[][];
+
 /**
  * Given a number t in the range [0,1], returns the corresponding color from the “RdYlGn” diverging color scheme represented as an RGB string.
  *
  * @param value Number in the range [0, 1].
  */
 export function interpolateRdYlGn(value: number): string;
+
+export const schemeRdYlGn: string[][];
+
 /**
  * Given a number t in the range [0,1], returns the corresponding color from the “Spectral” diverging color scheme represented as an RGB string.
  *
  * @param value Number in the range [0, 1].
  */
 export function interpolateSpectral(value: number): string;
 
+export const schemeSpectral: string[][];
+
 // -----------------------------------------------------------------------
 // Sequential
 // -----------------------------------------------------------------------
 /**
+ * Sequential, single-hue color schemes are available as continuous interpolators (often used with d3.scaleSequential) and
+ *  as discrete schemes (often used with d3.scaleOrdinal). Each discrete scheme, such as d3.schemeBlues, is represented as
+ *  an array of arrays of hexadecimal color strings. The kth element of this array contains the color scheme of size k; for example,
+ *  d3.schemeBlues[9] contains an array of nine strings representing the nine colors of the blue sequential color scheme.
+ *  Sequential, single-hue color schemes support a size k ranging from 3 to 9.
```

Comment:
Same as the above comment regarding the divergent scales: this comment only applies to the `interpolateBlues` function.